server/hedley/modules/custom/hedley_restful/plugins/restful/node/HedleyRestfulPrenatalEncounter.class.php

```diff
@@ -35,14 +35,14 @@ protected function alterQueryForViewWithDbSelect(SelectQuery $query) {
 
     $field_names = [
       'field_prenatal_encounter_type',
-      'field_prenatal_diagnosis',
+      'field_prenatal_diagnoses',
     ];
 
     foreach ($field_names as $field_name) {
       hedley_general_join_field_to_query($query, 'node', $field_name, FALSE);
     }
 
-    $query->addExpression("GROUP_CONCAT(DISTINCT field_prenatal_diagnosis.field_prenatal_diagnosis_value)", 'field_prenatal_diagnosis');
+    $query->addExpression("GROUP_CONCAT(DISTINCT field_prenatal_diagnoses.field_prenatal_diagnoses_value)", 'field_prenatal_diagnoses');
```

Comment:
@anvmn you probably did a full search and replace here, is this column present in the database? Or we should use the original value and keep the alias?

Reply:
@mariano-dagostino 
Field name itself was changed.
As this development is new, and was not deployed yet, column at DB will be created with new name.